pkgs/development/libraries/qt-5/5.12/default.nix

```diff
@@ -205,6 +205,7 @@ let
       qtwayland = callPackage ../modules/qtwayland.nix {};
       qtwebchannel = callPackage ../modules/qtwebchannel.nix {};
       qtwebengine = callPackage ../modules/qtwebengine.nix {
+        python = python2;
```

Comment:
This should be a good reason for accelerating https://github.com/NixOS/nixpkgs/issues/201765

Reply:
What's going to break if we just mark it broken here?

Reply:
When python2 gets marked as insecure, these will also drop anyway (by default).